src/textual/constants.py

```diff
@@ -37,3 +37,12 @@ def get_environ_bool(name: str) -> bool:
 
 LOG_FILE: Final[str | None] = get_environ("TEXTUAL_LOG", None)
 """A last resort log file that appends all logs, when devtools isn't working."""
+
+
+DEVTOOLS_PORT_ENVIRON_VARIABLE = "TEXTUAL_CONSOLE_PORT"
+DEFAULT_DEVTOOLS_PORT = 8081
+try:
```

Comment:
Let's factor this out in to a function. Something like `get_environment_int(env_var, default)`

Reply:
I pondered whether the `default` could be set to `None` (to return `None`) if the env var wasn't set but then decided against it because it would take an `overload` to ensure consistent typing and that seemed like overkill for this situation.

Reply:
I agree. These constants should have simple typing.